roofit/roofitcore/test/testRooFuncWrapper.cxx

```diff
@@ -327,3 +327,39 @@ TEST(RooFuncWrapper, GaussianNormalized)
       EXPECT_NEAR(getNumDerivative(*pdf, static_cast<RooRealVar &>(*paramsGauss[i]), normSet), dMyGauss[i], 1e-8);
    }
 }
+
+TEST(RooFuncWrapper, GaussianNormalizedAutoCodeSquash)
+{
+   using namespace RooFit;
+
+   RooRealVar x("x", "x", 0, -10, 10);
```

Comment:
```suggestion
   RooRealVar x("x", "x", 0, -10, std::numeric_limits<double>::infinity());
```
I was just thinking, now that we know the handling of infinites can be delicate, should be change the limits to include both a finite value and infinity? Like this, we would test both the correct differentiation of the `Erfc()`, and the correct handling of infinities (infinite limits for variables can happen regularly, so it's better to cover this with the test).